what4/CHANGES.md

```diff
@@ -9,14 +9,27 @@
 
 * Remove a dependency on `data-binary-ieee754`, which has been deprecated.
 
-* Deprecated `allSupported` which represents the SMT logic `ALL_SUPPORTED`,
+* Deprecate `allSupported` which represents the SMT logic `ALL_SUPPORTED`,
   and added `allLogic` instead which represents the SMTLib standard logic `ALL`.
```

Comment:
If you are switching to present tense, change "added" to "add"

Reply:
Oops, good catch.